### ISSUE TITLE
Weird formatting glitch

### ISSUE THREAD

**Issue by denis-sokolov** [scottcorgan/tap-spec#35]
This TAP input results in broken output.
```tap
TAP version 13
# comment
not ok 1 should be equal
  ---
    operator: equal
  ...
1..1
# tests 1
# pass  0
# fail  1
```

The output:
```
  comment


    ✗ should be equal
    ------------------
      0:
      1:
      2:
      3:
      4: o
      5: p
      6: e
      7: r
      8: a
      9: t
      10: o
      11: r
      12: :
      13:
      14: e
      15: q
      16: u
      17: a
      18: l





  Failed Tests: There was 1 failure

    comment

      ✗ should be equal


  total:     1
  passing:   0
  failing:   1
  duration:  16ms
```

I have generated the original input using npm `tape` and then minimized it manually for sanity.
I assume this is not intended.

**Comment by scottcorgan** [comment]
What's the code to produce that tap output?

**Comment by denis-sokolov** [comment]
```js
'use strict';
var tape = require('tape');
tape.test('simple test', function(t){
	t.equal(1, 2);
});
```

**Comment by herkyl** [comment]
:+1: Same problem for me

**Comment by grit96** [comment]
I have this problem too :+1:

**Comment by scottcorgan** [comment]
Published as `4.0.1`. Sorry about that.
